cli/script/release-hooks/signing.ts

```diff
@@ -19,6 +19,14 @@ interface CodeSigningClaims {
 
 var sign: cli.ReleaseHook = (currentCommand: cli.IReleaseCommand, originalCommand: cli.IReleaseCommand, sdk: AccountManager): q.Promise<cli.IReleaseCommand> => {
     if (!currentCommand.privateKeyPath) {
+        if (fs.lstatSync(currentCommand.package).isDirectory()) {
+            // If new update wasn't signed, but signature file for some reason still appears in the package directory - delete it
+            let signatureFilePath = path.join(currentCommand.package, METADATA_FILE_NAME);
+            if (fs.existsSync(signatureFilePath)) {
+                fs.unlinkSync(signatureFilePath);
```

Comment:
Don't we need here to check for access rights here to delete file? Similar as this is done below for removing the signature? If yes - does it make sense to use utility function for this?

Reply:
@max-mironov I think we don't need to do that. If something wrong with the access rights EPERM will be thrown and command execution will be stopped.

Reply:
@ruslan-bikkinin 
Probably it could be a good idea to show human friendly message like e.g. this one: https://github.com/Microsoft/code-push/blob/master/cli/script/release-hooks/signing.ts#L57 so users will be aware what's wrong here rather than throwing `EPERM ` exception. What do you think about this?

Reply:
@max-mironov Well, may be. I can do it.

Reply:
@ruslan-bikkinin, why don't you just wrapped up the following code into the method and used it for deleting `METADATA_FILE_NAME`? I don't think if it's a good idea to use different versions of the code/implementations to achieve the same goals.
```js
             var prevSignatureExists = true;
             try {
                 fs.accessSync(signatureFilePath, fs.F_OK);
             } catch (err) {
                 if (err.code === "ENOENT") {
                     prevSignatureExists = false;
                 } else {
                     return q.reject<string>(new Error(
                         `Could not delete previous release signature at ${signatureFilePath}.
                         Please, check your access rights.`)
                     );
                 }
             }
 
             if (prevSignatureExists) {
                 console.log(`Deleting previous release signature at ${signatureFilePath}`);
                 rimraf.sync(signatureFilePath);
             }
```